.travis.yml

```diff
@@ -16,7 +17,24 @@ install:
 
   # Install iris-grib's dependencies
   # --------------------------------
-  - conda install --quiet --yes python=${python} iris python-eccodes numpy cartopy mock filelock pep8;
+  # If running in Python 2, install Python2-only python-eccodes.
+  - if [[ "$python" == "2.7" ]]; then
+        export ECCODES_DEPS="python-eccodes";
+    fi
+
+  # If running in Python 3, we will install Python3-only eccodes-python via pip.
+  - if [[ "$python" == "3.6" ]]; then
+        export ECCODES_DEPS="eccodes pip";
+    fi
+
+  - conda install --quiet --yes python=${python} iris numpy cartopy mock filelock pep8 $ECCODES_DEPS;
```

Comment:
@lbdreyer Me being a shell snob - but could you make this `${ECCODES_DEPS}` please? :wink: